pages/common/history.md

```diff
@@ -7,6 +7,14 @@
 
 `history`
 
+- Run the n-th command of the history list:
```

Comment:
```suggestion
- Run the Nth command of the history list:
```

Reply:
Ok! I gonna fix it :)

Reply:
Actually, since all our clients support UTF-8, why not "Nᵗʰ"?

Reply:
For me Nᵗʰ is ok too. Should I change to it? @bl-ue @CleanMachine1

Reply:
I'm not sure anymore. In monospace, which terminals use, it looks like this: `Nᵗʰ`

Reply:
hmmm I like the smaller versions however as mentioned, special characters like that often look awful under most fonts.